vllm/distributed/parallel_state.py

```diff
@@ -58,10 +58,17 @@ def init_distributed_environment(
     local_rank: int = -1,
     backend: str = "nccl",
 ):
+    logger_data = {
+        "world_size": world_size,
+        "rank": rank,
+        "local_rank": local_rank,
+        "distributed_init_method": distributed_init_method,
+        "backend": backend,
+    }
     logger.debug(
         "world_size=%d rank=%d local_rank=%d "
         "distributed_init_method=%s backend=%s", world_size, rank, local_rank,
-        distributed_init_method, backend)
+        distributed_init_method, backend, extra=logger_data)
```

Comment:
I don't know how this function is used, but this could definitely be a place where we don't want to include `logger_data` since the log level of the message is debug. There's no way to avoid the object allocation and population and in most cases the log message will not be seen.